pkg/signing/signing.go

```diff
@@ -495,6 +495,7 @@ func getSignEntities(fimg *sif.FileImage) ([]string, error) {
 func GetSignEntities(cpath string) ([]string, error) {
 	fimg, err := sif.LoadContainer(cpath, true)
 	if err != nil {
+		fimg.UnloadContainer()
```

Comment:
Wont `sif.LoadContainer()` automatically call `fimg.UnloadContainer()` if theres an error? If not, should it do that?

Reply:
Oh, it seems like its not automatically called, https://github.com/sylabs/sif/issues/58. Then should this be fixed in the `sif` repo?

Reply:
@ikaneshiro has mentioned it might be a bit much to cut a new release of SIF for this, so it can be handled here for now... but can be fixed upstream in SIF later.